client/landing/gutenboarding/onboarding-block/vertical-background.tsx

```diff
@@ -17,10 +17,12 @@ import { STORE_KEY } from '../stores/onboard';
 import defaultImageUrl from '../../../../static/images/verticals/default.jpg';
 
 export interface VerticalBackgroundProps {
-	onLoad: () => void;
+	onLoad?: () => void;
```

Comment:
Does it make sense to keep this prop even if it's not used?

Reply:
I wasn't sure, to be honest 🙂 Do you think it's likely to be useful, or shall we just remove it?

Reply:
It's not very useful if we think from the React architecture perspective. If we need to do something at a certain moment in the lifecycle of a component, we should handle it inside the component. So I'd remove it for now and re-add later if there will be a use case.

Reply:
YAGNI :grimacing:

Reply:
> YAGNI 😬

Your favorite card 🙂 

Let's remove it 👍